packages/nodes-base/nodes/RabbitMQ/RabbitMQ.node.ts

```diff
@@ -478,6 +477,8 @@ export class RabbitMQ implements INodeType {
 
 				const exchangePromises = [];
 				for (let i = 0; i < items.length; i++) {
+					const routingKey = this.getNodeParameter('routingKey', i) as string;
+
```

Comment:
Hello @ricardokenji , thanks for the PR, we need to version this change as it could be breaking for existing workflows